### PR TITLE
Update AlphaNet

### PR DESCRIPTION
**Pull request by zmyybc**
## Description

Hi!
I am submitting 2 v1 AlphaNet models:
- AlphaNet-v1-MPtrj: compiled model trained on MPtrj
- AlphaNet-v1-OMA: Pretrained on OMAT24, finetuned on sALEX+MPtrj

Appreciate your work again!

## Checklist

Please check the following items before submitting your PR:

- [x] I created a new folder and YAML metadata file `models/<arch_name>/<model_variant>.yml` for my submission. `arch_name` is the name of the architecture and `model_variant.yml` includes things like author details, training set names and important hyperparameters.
- [x] I added the my new model as a new attribute on the [`Model.<arch_name>` enum](https://github.com/janosh/matbench-discovery/blob/57d0d0c8a14cd317/matbench_discovery/enums.py#L274) in `enums.py`.
- [x] I uploaded the energy/force/stress model prediction file for the WBM test set to Figshare or another cloud storage service (`<yyyy-mm-dd>-<model_variant>-preds.csv.gz`).
- [x] I uploaded the model-relaxed structures file to Figshare or another cloud storage service in [JSON lines format](https://jsonlines.org) (`<yyyy-mm-dd>-wbm-IS2RE-FIRE.jsonl.gz`). JSON Lines allows fast loading of small numbers of structures with `pandas.read_json(lines=True, nrows=100)` for inspection.
- [x] I uploaded the phonon predictions to Figshare or another cloud storage service (`<yyyy-mm-dd>-kappa-103-FIRE-<values-of-dist|fmax|symprec>.gz`).
- [x] I included the urls to the Figshare files in the YAML metadata file (`models/<arch_name>/<model_variant>.yml`). If not using Figshare I have included the urls to the cloud storage service in the description of the PR.
- [x] I included the test script (`test_<arch_name>_<task>.py` for `task` in `discovery`, `kappa`, `diatomics`) that generated the prediction files.

## Additional Information (Optional)

- [ ] I included a training script (`train_<arch_name>.py`) if I trained a model specifically for this benchmark.
- [ ] I included a `readme.md` with additional details about my model.

### CODE REVIEW

**Comment by janosh** [comment]
thanks @zmyybc! i'll try to take a look tomorrow

**Comment by zmyybc** [comment]
ok

**Review by janosh — APPROVED** [review]
thanks a lot for this submission @zmyybc! 👍 

apologies for the long wait for a review. nothing to do with the model, just bad time management on my part
